service/history/historyReplicator.go

```diff
@@ -37,6 +38,9 @@ import (
 	"github.com/uber/cadence/common/persistence"
 )
 
+var errRetry = &shared.RetryTaskError{Message: "History replicator retry error"}
```

Comment:
Can you define all errors within the same var section.  There is already one error for RetryTaskError.